Files/UserControls/Widgets/DrivesWidget.xaml.cs

```diff
@@ -24,6 +26,24 @@ public DrivesWidget()
             InitializeComponent();
         }
 
+        private async void EjectDevice_Click(object sender, RoutedEventArgs e)
+        {
+            var item = ((MenuFlyoutItem)sender).DataContext as DriveItem;
```

Comment:
What about to do the same in SidebarControl.xaml.cs?

Reply:
@tsvietOK We already do.

Reply:
It's a little out of scope for this pull request but we can.